invisible_cities/core/core_functions.py

```diff
@@ -59,7 +59,7 @@ def relative_difference(x, y, *, norm_mode=NormMode.first):
         raise TypeError(f"Unrecognized relative difference option: {norm_mode}")
 
 
-def in_range(data, minval=-np.inf, maxval=np.inf):
+def in_range(data, minval=-np.inf, maxval=np.inf, left="close", right="open"):
```

Comment:
Indeed, it should be "closed". I thought of an enum, but I think it can make it uncomfortable to use since this function is not so much used in the cities, but externally. On the other hand this might be a lazy-person argument... I am open to change it.

Reply:
I've pushed a version using an enum to compare and make a decision on this.

Reply:
I like that now you have a test to show the valid values in the Enum case so I'd use this version as long as the other developers don't find it too clunky.

Reply:
I think I'd prefer something like `left=True, right=False`, that way the user will not have to import enum class and is somehow consistent with numpy/pandas style?

Reply:
I agree with @mmkekic to have as arguments: left_closed = True, right_closed = False